contracts/LockDealBundleProvider/LockDealBundleProvider.sol

```diff
@@ -109,7 +113,12 @@ contract LockDealBundleProvider is
     ) public onlyProvider {
     }
 
-    function getData(uint256 poolId) public view override onlyBundlePoolId(poolId) returns (IDealProvierEvents.BasePoolInfo memory poolInfo, uint256[] memory params) {
+    function getData(uint256 poolId) public view override returns (IDealProvierEvents.BasePoolInfo memory poolInfo, uint256[] memory params) {
+        require(
+            poolIdToLockDealBundle[poolId].firstSubPoolId > 0,
+            "Pool is not a bundle pool"
+        );
```

Comment:
this is view function. no need to check this.
if it have no data, it will show 0